src/common/state/error_table.h

```diff
@@ -18,14 +18,19 @@ typedef enum {
   /** An object was added with a different hash from the existing
```

Comment:
Yeah, there are several instances of having to define a new struct just to access it in `redis.cc`...